netgrasp/utils/mac.py

```diff
@@ -0,0 +1,255 @@
+from __future__ import print_function
+
+import os
+import re
+import datetime
+
+TEST_WSDATA = False
+TEST_WSDATA_FILENAME = '~/.netgrasp.mac.wsdata'
+
+DB_LOOKUP_TABLE_NAME = 'macvendor'
+DB_LOOKUP_CONFIG_SOURCE_KEY = 'maclookup_source'
+DB_LOOKUP_CONFIG_UPDATED_KEY = 'maclookup_updated'
+
+LOOKUP_SOURCES = ['local', 'remote']
+
+DEFAULT_LOOKUP_SOURCE = 'remote'
```

Comment:
I'm wondering if we can instead check if we have local data and if so use it; otherwise use remote data. (We'd only have to check once, or once every thirty minutes or something if we want to switch to local once downloaded)